settings.yml

```diff
@@ -84,3 +84,12 @@ custom_headers:
         href (optional): valid href</br>
       Example: Teams,links:below,/teams adds the header "Teams" to the custom links that appear
       below the category list
+child_categories_default_state:
```

Comment:
I think perhaps we should break out the ``hidden`` option into a different boolean setting as it's not really a "default state", as with that on children will never appear.